PWGLF/NUCLEX/Hypernuclei/Hyp3Body/AliAnalysisTaskHypertriton3ML.cxx

```diff
@@ -455,6 +486,23 @@ void AliAnalysisTaskHypertriton3ML::UserExec(Option_t *) {
 
         hyp3r.fIsMatter = deu->Charge() > 0;
 
+        if (fApplyML) {
+          auto fMap = FeaturesMap(hyp3r, fREvent);
+
+          float ct = fMap["ct"];
+          float invmass = fMap["InvMass"];
+          float hyppt = fMap["HypCandPt"];
+
+          if (ct > 1. && ct < 35. && fREvent.fCent < 90.) {
+            double score;
+            if (fMLResponse->IsSelected(ct, fMap, &score)) {
+              MLSelected hyp3ml{score, invmass, ct, fREvent.fCent, hyppt};
```

Comment:
This is not compiling on Mac:

/Users/markusfasel/alice/sw/SOURCES/AliPhysics/ali-master/0/PWGLF/NUCLEX/Hypernuclei/Hyp3Body/AliAnalysisTaskHypertriton3ML.cxx:499:33: error: non-constant-expression cannot be narrowed from type 'double' to 'float' in initializer list [-Wc++11-narrowing]
              MLSelected hyp3ml{score, invmass, ct, fREvent.fCent, hyppt};
                                ^~~~~
/Users/markusfasel/alice/sw/SOURCES/AliPhysics/ali-master/0/PWGLF/NUCLEX/Hypernuclei/Hyp3Body/AliAnalysisTaskHypertriton3ML.cxx:499:33: note: insert an explicit cast to silence this issue
              MLSelected hyp3ml{score, invmass, ct, fREvent.fCent, hyppt};
                                ^~~~~
                                static_cast<float>( )

Reply:
Ups, sorry... this will fix it https://github.com/alisw/AliPhysics/pull/12638